src/sentry/search/snuba/executors.py

```diff
@@ -263,103 +236,90 @@ def snuba_search(
         else:
             # Get the top matching groups by score, i.e. the actual search results
             # in the order that we want them.
-            orderby = [
-                f"-{sort_field}",
-                "group_id",
-            ]  # ensure stable sort within the same score
+            orderby = [f"-{sort_field}", "group_id"]  # ensure stable sort within the same score
             referrer = "search"
 
-        query_partial = functools.partial(
-            snuba.aliased_query,
-            dataset=snuba.Dataset.Discover,
-            start=start,
-            end=end,
-            selected_columns=selected_columns,
-            groupby=["group_id"],
-            limit=limit,
-            offset=offset,
-            orderby=orderby,
-            referrer=referrer,
-            having=having,
-            filter_keys=filters,
-            totals=True,  # Needs to have totals_mode=after_having_exclusive so we get groups matching HAVING only
-            turbo=get_sample,  # Turn off FINAL when in sampling mode
-            sample=1,  # Don't use clickhouse sampling, even when in turbo mode.
+        query_partial: SearchQueryPartial = cast(
+            SearchQueryPartial,
+            functools.partial(
+                aliased_query_params,
+                dataset=snuba.Dataset.Discover,
```

Comment:
It might make sense for the dataset to be configurable per category as well. For future issue types I don't think we'll be able to use Discover

Reply:
Yeah, that makes sense.